docs/design/chart_format.md

```diff
@@ -69,16 +66,16 @@ We assume that we are developing a technology that will enjoy widespread use amo
 Based on these, our design goal is to make it **easy to find, use, author, release, and maintain charts**, while still making it possible for advanced users to build sophisticated charts. Furthermore, we have begun the process of **attaching provenance to charts**.
 
 
-We have attempted to build a proposal that does not lose any of the functionality of DM, and which loses only Helm functionality that we believe is disposable. But both tools will need to change substantial portions of their chart-handling logic.
-### Non-Goals
-This proposal doesn’t define how either the client or the server must use this data, though we make some non-normative recommendations (e.g. “this data may be displayed to the user” or “a suitable backend will be able to translate this to a fully qualified URL”).
+This design does not lose any of the functionality of DM, and which loses only Helm functionality that we believe is disposable. But both tools have changed substantial portions of their chart-handling logic.
```

Comment:
Since this is the first time the acronym `DM` is used, the meaning should be spelled out, and the acronym placed in parentheses, like this:
```
Deployment Manager (DM)
```